nat-lab/docker-compose.yml

```diff
@@ -71,8 +71,7 @@ services:
     networks:
       cone-net-01:
         ipv4_address: 192.168.101.104
-    ports:
-      - "58001:58001"
+    ports: ["58001"]
```

Comment:
Maybe add some comment warning that this part is modified by python, and link to the place

Reply:
Done